docs/InstallEbpf.md

```diff
@@ -16,8 +16,8 @@ This method uses a machine that
 has already built the binaries for x64/Debug or x64/Release.
 
 1. Deploy the binaries to `C:\Temp` in your VM, as follows:
-    a. If you built the binaries from inside the VM, then from your ebpf-for-windows directory in the VM, do `.\scripts\deploy-ebpf -l`.  Otherwise,
-    b. If you built the binaries on the host machine, then from your ebpf-for-windows directory on the host machine, start an admin Powershell on the host machine and do `.\scripts\deploy-ebpf`, or to also copy files needed to run various tests, do `.\scripts\deploy-ebpf -t`.
+    a. If you built the binaries from inside the VM, then from your ebpf-for-windows directory in the VM, do `.\x64\debug\deploy-ebpf -l`.  Otherwise,
```

Comment:
I don't like having to change the path here that users use to run scripts.
I'd suggest keeping deploy-ebpf.ps1 where it is and instead generating a helper psm1 file in x64\debug that can be invoked from deploy-ebpf.ps1

Reply:
On second thought, your change enables deploying debug vs release separately in a way that is explicit, so ok